### DISCUSSION TITLE
Hide starred repos that have been added to a list

### DISCUSSION THREAD

**Discussion by srivishp** [community/community#61968] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hide the repos that have been added to a list in the Stars section. So that the Stars section contains only unlisted repos.

**Comment by depascalis** [comment]
I agree, BUT, there must be a view that lists all starred repos. It could be a default list where all starred repos are automatically added for example.

**Reply by SASCHA00000** [reply]
# - [ ] ![icon.txt](https://github.com/community/community/files/12517213/icon.txt)

**Reply by salmanasiddiqui** [reply]
I think filtering starred repos with lists is needed. With a filtering option to view starred repos not part of any list.

**Reply by Hi2885** [reply]
> I think filtering starred repos with lists is needed. With a filtering option to view starred repos not part of any list.

**Comment by tanxinzhe28** [comment]
Thanks you so much

**Comment by Lugo778** [comment]
> - -

**Comment by Azeemahx2** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Hide the repos that have been added to a list in the Stars section. So that the Stars section contains only unlisted repos.

**Comment by felixxxxxs** [comment]
This would make the lists feature even more useful, as I'd like to be able to organize my starred repos in different lists. Repos that are in a specific list already don't need to be shown in the general "starred repos" list. Bump!

**Comment by ksharizard** [comment]
Hi! I was facing the same issue with having a lot of stars and not knowing which are unlisted. I created a bash script to get a user's stars and lists and find the unlisted stars. I tested this on my profile and it worked well to output the unlisted repos. Just linking it here if anybody finds it interesting and wants to try it out:

https://github.com/ksharizard/github-lists

**Comment by devinhurry** [comment]
I created a chrome extension to show unlisted stars. 
https://github.com/devinhurry/github-star-show-unlisted
